### ISSUE TITLE
tf.keras.preprocessing.image_dataset_from_directory does not work in Keras 2.4.2

### ISSUE THREAD

**Issue by garikhgh** [keras-team/keras#14132]
Hi, could you explain.

this does not work

tf.keras.preprocessing.image_dataset_from_directory(
    directory,
    labels="inferred",
    label_mode="int",
    class_names=None,
    color_mode="rgb",
    batch_size=32,
    image_size=(256, 256),
    shuffle=True,
    seed=None,
    validation_split=None,
    subset=None,
    interpolation="bilinear",
    follow_links=False,
)

this is taken from keras webpage

**Comment by fchollet** [comment]
Install TF-nightly and it will work. `pip install tf-nightly`

**Comment by garikhgh** [comment]
ok thanks
